package.json

```diff
@@ -37,6 +37,7 @@
     "webpack": "^3.11.0"
   },
   "scripts": {
-    "build": "webpack"
+    "start": "jekyll serve --incremental --watch | webpack -w",
```

Comment:
Jekyll pipes output to webpack?

Reply:
Looking into this I think the pipe is actually not correct. It seems the act of piping may have the side effect of working as expected but it isn't intended. webpack does have a `--watch-stdin` I think for this purpose however, after googling for CLI documentation on webpack (hint: there seem to be none) I found this blog post that is doing exactly what your looking for:

https://www.jonathan-petitcolas.com/2016/08/12/plugging-webpack-to-jekyll-powered-pages.html

And they use a program called concurrency which is installed via npm (yarn  in our case). I think that looks like a much cleaner solution then to rely on shell file handler mechanics to simulate job control mechanics.